### PR TITLE
Adding cloudwatch exporter repository

### PR DESCRIPTION
**Pull request by iw-gtadam**

### CODE REVIEW

File: docker-cloudwatch-exporter.tf [hunk 1]
```diff
@@ -0,0 +1,66 @@
+resource "github_repository" "cloudwatch_exporter" {
+  name             = "cloudwatch-exporter"
```
Comment (dan-hill2802) [inline docker-cloudwatch-exporter.tf:2]: maybe 
```suggestion
  name             = "prometheus-cloudwatch-exporter"
```

File: docker-cloudwatch-exporter.tf [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  description      = "cloudwatch-exporter"
```
Comment (dan-hill2802) [inline docker-cloudwatch-exporter.tf:3]: can you give it a proper description?

File: docker-cloudwatch-exporter.tf [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  }
+
+  template {
+    owner = "${var.github_organization}"
+    repository = "dataworks-repo-template-docker"
+  }
+}
+
+resource "github_team_repository" "cloudwatch_exporter_dataworks" {
+  repository = "${github_repository.cloudwatch-exporter.name}"
```
Comment (dan-hill2802) [inline docker-cloudwatch-exporter.tf]: should be underscores (repeats in several places)

File: docker-cloudwatch-exporter.tf (hunk 2 above)
Comment (iw-gtadam) [reply docker-cloudwatch-exporter.tf:3]: Sorry hadn't raised it yet, was still reviewing myself, but of course sure.

File: docker-cloudwatch-exporter.tf (hunk 3 above)
Comment (iw-gtadam) [reply docker-cloudwatch-exporter.tf]: As mentioned had not raised it yet, in fact your comment comes after the fix had been put in place

File: docker-cloudwatch-exporter.tf (hunk 1 above)
Comment (iw-gtadam) [reply docker-cloudwatch-exporter.tf:2]: Don't disagree with this but we already have another prometheus application, alertmanager. That does not begin with prometheus and I would like them to be consistent whichever way we go. Are you attached to it beginning with prometheus?

Comment (dan-hill2802) [reply docker-cloudwatch-exporter.tf:2]: my concern is when we have to implement other CloudWatch exporters, like the one for Splunk

**Pull request merged**